File: problemas/1000/1061.cs
using System;
using System.Globalization;

namespace uri1061
{
    internal static class Program
    {
        private static void Main()
        {
            string diaInicio = Console.ReadLine().Replace("Dia ", "");
            string horaInicio = Console.ReadLine().Replace(" ", "");
            string diaFim = Console.ReadLine().Replace("Dia ", "");
            string horaFim = Console.ReadLine().Replace(" ", "");

            var dataInicio = Convert.ToDateTime($"04-{diaInicio}-{DateTime.Now.Year} {horaInicio}",
                CultureInfo.InvariantCulture);
            var dataFim =
                Convert.ToDateTime($"04-{diaFim}-{DateTime.Now.Year} {horaFim}", CultureInfo.InvariantCulture);
            var duracao = dataFim - dataInicio;

            Console.WriteLine($"{duracao.Days} dia(s)");
            Console.WriteLine($"{duracao.Hours} hora(s)");
            Console.WriteLine($"{duracao.Minutes} minuto(s)");
            Console.WriteLine($"{duracao.Seconds} segundo(s)");
        }
    }
}

File: problemas/1000/1061.php
<?php

$diaInicio = str_replace('Dia ', '', readline());
$horaInicio = str_replace(' ', '', readline());
$diaFim = str_replace('Dia ', '', readline());
$horaFim = str_replace(' ', '', readline());

$ano = date_create('NOW')->format('Y');
$dataInicio = new DateTime("{$ano}-04-{$diaInicio} {$horaInicio}");
$dataFim = new DateTime("{$ano}-04-{$diaFim} {$horaFim}");
$duracao = date_diff($dataFim, $dataInicio);

echo $duracao->format("%d dia(s)\n");
echo $duracao->format("%h hora(s)\n");
echo $duracao->format("%i minuto(s)\n");
echo $duracao->format("%s segundo(s)\n");